presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatisticType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.statistics;
+
+public enum ColumnStatisticType
```

Comment:
not necessarily. SPI is a common denominator, something we ingest. Here we have superset of all possible stats external systems can store. 
SPI says want we want for CBO.
Here we say what hive need (or other programs using metastore may need, including ourselves)

Reply:
@rschlussel There is no exact match. `NUMBER_OF_TRUE_VALUES` is something boolean specific, `NUMBER_OF_NON_NULL_VALUES` is chosen over `NUMBER_OF_NULL_VALUES` just because it is easier to compute (no extra projection), and so on.